web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/utils.py

```
@@ -0,0 +1,332 @@
+            server['username'],
+            server['db_password'],
+            server['host'],
+            server['port'],
+            server['sslmode']
+        )
+        pg_cursor = connection.cursor()
+        pg_cursor.execute(
+            f"SELECT array_agg(a.attname) FROM pg_catalog.pg_attribute a "
+            f"JOIN pg_catalog.pg_statistic_ext s ON a.attrelid = s.stxrelid "
+            f"WHERE s.oid = {statistics_oid} "
+            f"AND a.attnum = ANY(s.stxkeys)"
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)